### DISCUSSION TITLE
Feature request - Name + Title on chat bubble?

### DISCUSSION THREAD

**Discussion by Androrier** [flyerhq/flutter_chat_ui#108] [Ideas & Requests]
For example:
![Screen Shot 2021-09-13 at 12 39 01](https://user-images.githubusercontent.com/81566467/133061639-456c613f-7a48-4509-9bb3-9ccf7065c355.png)

If were to implement it, will it be merged?
Thank you.

**Comment by demchenkoalex** [comment]
I guess that's overlapping with [this](https://github.com/flyerhq/flutter_chat_ui/discussions/102) discussion?
